crates/proto/src/xfer/dnssec_dns_handle.rs

```diff
@@ -262,13 +261,12 @@ async fn verify_rrsets<H>(
     handle: DnssecDnsHandle<H>,
     records: Vec<Record>,
     options: DnsRequestOptions,
-    current_time: u32,
 ) -> Vec<Record>
 where
     H: DnsHandle + Sync + Unpin,
 {
     let mut rrset_types: HashSet<(Name, RecordType)> = HashSet::new();
-    let mut rrset_proofs: HashMap<(Name, RecordType), Proof> = HashMap::new();
+    let mut rrset_proofs: HashMap<(Name, RecordType), (Proof, Option<u32>)> = HashMap::new();
```

Comment:
Probably not for this PR, but I think it would really help to define a `struct UnixTime(u32)` to make the type more legible.

Reply:
Good idea, that is what @japaric suggested in [their comment](https://github.com/hickory-dns/hickory-dns/pull/2311#discussion_r1679068157).